packages/docs/src/styles/landing.css

```
@@ -2,6 +2,83 @@
    Maina Landing Page — Dark Terminal Aesthetic
    ============================================ */
 
+/* ─── Tailwind 4 @theme (feature 050 design tokens) ──────────────────
+   New components read these via Tailwind utilities (`bg-bg`,
+   `text-ink`, etc). Legacy `:root` vars below are kept until the
+   old components (Features.astro et al) are deleted in Phase 9. */
+
+@theme {
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)